### PR TITLE
Added Vercel.com to Web Hosting

### PR DESCRIPTION
**Pull request by zxsleebu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -692,6 +692,7 @@ Table of Contents
   * [surge.sh](https://surge.sh/) — Static web publishing for Front-End developers. Unlimited sites with custom domain support
   * [tilda.cc](https://tilda.cc/) — One site, 50 pages, 50 MB storage, only the main pre-defined blocks among 170+ available, no fonts, no favicon and no custom domain
   * [txti.es](https://txti.es/) — Quickly create web pages with markdown.
+  * [Vercel (Zeit)](https://vercel.com) — Build, deploy, host apps directly from GitHub repository
```
Comment (ripienaar) [inline README.md:695]: It's already listed under PaaS

Comment (styfle) [inline README.md:695]:
```suggestion
  * [Vercel](https://vercel.com) — Build and deploy web apps with `git push`, perfect for Next.js
```

**Review by styfle — APPROVED** [review]

**Comment by ripienaar** [comment]
It's already listed under PaaS, won't be adding it twice.

**Comment by styfle** [comment]
Would you consider moving it from PaaS to Web Hosting?

**Comment by ripienaar** [comment]
yup, feel free to do a PR to move it

**Pull request closed without merging**